addons/mail/static/src/components/notification_list/notification_list.xml

```diff
@@ -39,7 +39,7 @@
                             />
                         </t>
                         <t t-if="!notificationView_last">
-                            <div class="o_NotificationList_separator border-bottom"/>
+                            <div class="o_NotificationList_separator w-100 flex-shrink-0 border-bottom"/>
```

Comment:
According to the SCSS coding guidelines, `flex-shrink-0` should be before `w-100`.
https://github.com/odoo/odoo/wiki/SCSS-coding-guidelines